tasks/build-website-development.js

```diff
@@ -0,0 +1,8 @@
+var gulp = require('gulp');
+var config = require('./config.js');
+
+gulp.task('build-website-development', ['prepare-website'], buildWebsiteEevelopmentTask);
+
+function buildWebsiteEevelopmentTask (cb) {
```

Comment:
I suggest named functions, lets suppose you dont have a variable config, or another error inside that function, if is a anonymous function, the error will be simillar to:

```
error on anonimous function in file.js
```

now if we name the function

```
error on 'methodName' in file.js
```